### PR TITLE
DEVELOPER-2914 Hid Devnation discount content before login

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by LightGuard** [comment]
Maybe I'm missing something, but um, is simply hiding it enough? Seems like we should be removing it.

**Comment by adelasofia** [comment]
@LightGuard we are only hiding it until the user registers then it is shown

**Comment by LightGuard** [comment]
Okay, fair enough

**Comment by paulrobinson** [comment]
I think it would be better if the code was hidden by default. There's still a small amount of time when the page loads, that you can see it: https://www.dropbox.com/s/xk6aq4l6ceubilr/Screenshot%202016-04-28%2008.49.13.png?dl=0. Although, I'm not sure this is possible with slim?

I think we can merge this as an improvement over what is in production, once @MikGue approves.

File: promotions/devnation-discount-code.html.slim [hunk 1]
```diff
@@ -20,9 +20,10 @@ drupal_type: rhd_microsite
         br
         | To register for DevNation 2016 and get the 30% discount, click
         a(href="https://rh2016.smarteventscloud.com/portal/createAccount.ww?")  here
-        |  and enter the passcode: “
-        strong DEVSAVE30
```
Comment (LightGuard) [inline promotions/devnation-discount-code.html.slim]: You could do:

    strong.hide DEVSAVE30

to hide it initially.

File: promotions/devnation-discount-code.html.slim [hunk 2]
```diff
@@ -20,9 +20,10 @@ drupal_type: rhd_microsite
...
-        |”.
+        |  and enter the passcode: 
+        strong.devnation-code DEVSAVE30
```
Comment (LightGuard) [inline promotions/devnation-discount-code.html.slim]: You could use `.hide` on the `strong` element to hide it initially.

**Comment by LightGuard** [comment]
Superseded by #1069

**Pull request closed without merging**